3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -10,10 +11,11 @@
 using System.Text;
 using System.Windows.Forms;
 using Color = System.Drawing.Color;
+using Utilities;
 
 namespace Max2Babylon
 {
-    internal partial class BabylonExporter
+    internal partial class BabylonExporter : ILoggingProvider
```

Comment:
This is probably one of the ugliest parts of this refactor, I'm not really a huge fan of adding this interface, but it was the easiest way to reconcile all the areas where we're logging in the GLTF converter. Maybe instead have a logging class that handles writing to the UI implementation?

Reply:
yup you could have a logger implementing the interface in both max and maya and then you could pass the interface in the exporter constructors ???